assets/css/tailwind.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
+  purge: {
+    enabled: true,
```

Comment:
This should be removed. `enabled: true` runs Purge in every instance instead of just production.

Reply:
I'll fix that. I assume that we can remove postcss-import as well with this change?

Reply:
It does not purge without `enabled: true`. Just tested that. Additionally when setting purge up this way:
`  purge: [
   './layouts/**/*.html',
  ],`

There are some issues with autoprefixer.

Reply:
No. `enabled: true` makes it purge every single time. It should only purge in production: 

> Now whenever you compile your CSS with NODE_ENV set to production, Tailwind will automatically purge unused styles from your CSS.

https://tailwindcss.com/docs/controlling-file-size/#enabling-manually